frontend/templates/troubleshooting/index.tsx

```diff
@@ -431,18 +469,20 @@ function NavTabs({
       ...baseStyleProps,
       borderColor: 'transparent',
       color: 'gray.500',
-      fontWeight: 400,
+      fontWeight: 'normal',
       sx: {
          '&:visited': {
             color: 'gray.500',
          },
-         '&:hover': {
-            textDecoration: 'none',
-         },
-         '&:hover:not(.isDisabled)': {
-            textDecoration: 'none',
-            color: 'gray.700',
-            background: 'gray.100',
+         '@media (hover: hover)': {
+            '&:hover': {
+               textDecoration: 'none',
+            },
+            '&:hover:not(.isDisabled)': {
+               textDecoration: 'none',
+               color: 'gray.700',
+               background: 'gray.100',
+            },
```

Comment:
Also not your fault, but I think we might want to use chakra's [groupHover](https://chakra-ui.com/docs/styled-system/style-props#pseudo)

Reply:
I really like these. TIL how extensive they are 🍻

Reply:
https://github.com/iFixit/react-commerce/pull/1816/commits/170549ba71521b796eb197cc8f47e5b41e52e0b8